src/heist_path/HeistPathB.py

```
@@ -142,41 +158,180 @@ def goto_lg1_skip(self):
-            self.send_key_down("w")
-            self.sleep(0.10)
-            self.wait_and_interact(direction="w", is_lock=False, time_out=3.65)
+            self.send_key("w", down_time=0.32)
             self.sleep(0.10)
+            self.send_key_down('d')
+            self.sleep(0.05)
+            self.wait_and_interact(direction="d", is_lock=False, time_out=3.65)
+            if self.find_interac():
+                is_open_door = self.lobby_open_door_check()
+                if not is_open_door:
+                    raise AbortException("timeout for wait_and_interact") # 考虑之后加复位或其他
```

Ruff (0.15.14)

[error] Undefined name `AbortException`

(F821)